deploy/crds/org_v1_che_crd.yaml

```
@@ -507,6 +507,11 @@ spec:
                     TLS mode. This is enabled by default. Disabling TLS may cause
                     malfunction of some Che components.
                   type: boolean
+                useServiceHostNames:
```

If you google hostname - it will be one word
```suggestion
                useServiceHostnames:
```

---

or just `serviceHostnames` ?

---

As for me both variants are Ok:  serviceHostnames and useServiceHostnames